pages/linux/wajig.md

```diff
@@ -0,0 +1,36 @@
+# wajig
+
+> Wajig is a simplified, all-in-one-place, system support tool for Debian-based OSes.
+> More information: <https://wajig.togaware.com>.
+
+- Update the list of available packages and versions:
+
+`wajig update`
+
+- Install a package, or update it to the latest available version:
+
+`wajig install {{package}}`
+
+- Remove a package and its configuration files:
+
+`wajig purge {{package}}`
+
+- Perform an update then a dist-upgrade:
+
+`wajig daily-upgrade`
+
+- Display installed sizes of installed packages:
+
+`wajig sizes`
+
+- List version and distribution of installed packages:
+
+`wajig versions`
+
+- List versions of upgradable packages:
```

Comment:
Output is identical as `apt-show-versions`:
```
zmap:amd64/stable 2.1.1-2+b6 uptodate
zmap:i386 not installed
zoom:amd64 5.6.13558.0321 installed: No available version in archive
```

Reply:
Right, but that doesn't answer my question @rozie. Is the version displayed the local version or the remote version? We should clarify this in the example description.

Reply:
It shows version only for installed packages (installed, removed, but present in local DB to be precise). But it shows both intstalled version, and version available in archive:
```
vlc:amd64/sid 1:3.0.12-dmo2 upgradeable to 1:3.0.16-dmo4
```
Version in archive is visible only after `apt update` command is issued, so it also uses local DB.

man of the apt-show-versions calls it:
"Lists available package versions with distribution"
help:
"shows available versions of installed packages"
and wajig calls it "list versions of upgradable packages"

I believe all of these short decritptions are correct and none is precise.

Reply:
I think in this case we can leave the description

Reply:
Right, ok. What about this?
```suggestion
- List both local and remote versions of upgradable packages:
```

Reply:
I guess this would be a bit misleading. Remote versions is displayed only if local DB is already aware of it.
I'd try to leave it as is at this moment and wait for users' feedback.